spec/models/mhv_account_spec.rb

```diff
@@ -164,170 +156,150 @@
     end
 
     context 'check_account_state' do
-      context 'with terms accepted' do
-        let(:terms) { create(:terms_and_conditions, latest: true, name: described_class::TERMS_AND_CONDITIONS_NAME) }
-        before(:each) { create(:terms_and_conditions_acceptance, terms_and_conditions: terms, user_uuid: user.uuid) }
-
-        context 'without an existing account' do
-          context 'nothing has been persisted and no mhv_id' do
-            it 'has no_account' do
-              expect(subject.account_state).to eq('no_account')
-              expect(subject.creatable?).to be_truthy
-              expect(subject.terms_and_conditions_accepted?).to be_truthy
-            end
+      context 'without an existing account' do
```

Comment:
This whole section was just moving everything out of the `with terms accepted` context block.